### PR TITLE
Sectioned cluster_fallback configs + added WebAssembly text to Wasm section

### PR DESCRIPTION
**Pull request by d19dotca**
### Description

Moved the DP Resilience feature (Cluster Fallback) properties to their own section for better organization and linking. Loosely related to https://github.com/Kong/docs.konghq.com/pull/6064.

Also noticed our Wasm section had no reference to the text 'WebAssembly' which resulted in it not being found easily. Added that to the title of that section to allow for better searchability.

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/gateway/reference/configuration.md [hunk 1]
```diff
@@ -4186,12 +4186,37 @@ and valid UUID. When empty, node ID is automatically generated.
 
 **Default:** none
 
+---
+
+## Cluster Fallback Configuration section
+
+
+### cluster_fallback_config_export
```
Comment (lena-larionova) [inline app/_src/gateway/reference/configuration.md:4194]: Based on https://github.com/Kong/kong-ee/blob/master/kong.conf.default#L3388-L3390, these entries should come after the other two in the list.

**Comment by lena-larionova** [comment]
Setting a "do not merge" for now, I've opened up a [parallel PR on the kong-ee side](https://github.com/Kong/kong-ee/pull/6432) to get these merged in + some cleanup/fixes to the cluster fallback section.

File: app/_src/gateway/reference/configuration.md (hunk 1 above)
Comment (d19dotca) [reply app/_src/gateway/reference/configuration.md:4194]: Ah I see, I was trying to make it alphabetical, I didn't fully realize it had to match the order from kong.conf, but that makes sense to me then. 👍

Comment (d19dotca) [reply app/_src/gateway/reference/configuration.md]: Made the correction now.

**Review by lena-larionova — APPROVED** [review]

**Comment by lena-larionova** [comment]
@d19dotca fyi, I also pulled in the edits I made on the source repo side.

**Pull request merged**